pkgs/applications/misc/feedbackd/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkgconfig
+, cmake
+, wrapGAppsHook
+, glib
+, gsound
+, libgudev
+, json-glib
+, vala
+, gobject-introspection
+}:
+
+stdenv.mkDerivation rec {
+  pname = "feedbackd";
+  version = "v0.0.0+git20200420";
+
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "Librem5";
+    repo = pname;
+    rev = "fd5e63c0a0c1b9296249d517c6849402a3a3ca10";
+    sha256 = "0glzc284wbvwvax52lp6sqr4whhpbqrkn8isidlqz1yrag3phfv9";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    meson
+    ninja
+    pkgconfig
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    glib
+    gsound
+    libgudev
+    json-glib
+    vala
+    gobject-introspection
+  ];
+
+  configurePhase = "meson build --prefix=$out";
```

Comment:
these phases should be handled properly thanks to `pkgs/development/tools/build-managers/meson/setup-hook.sh` and `pkgs/development/tools/build-managers/ninja/setup-hook.sh`.

Reply:
Commenting this lines result in:

```
building '/nix/store/llm9jqrmrxjai94zmixjabkqp58fcpdx-feedbackd-v0.0.0+git20200420.drv'...
unpacking sources
unpacking source archive /nix/store/gp03dsj195lq7mv0zn62ax6wjygacbfj-source
source root is source
patching sources
configuring
fixing cmake files...
cmake flags: -GNinja -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/share/doc/ -DCMAKE_INSTALL_INFODIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/include -DCMAKE_INSTALL_SBINDIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_OSX_ARCHITECTURES=x86_64 -DCMAKE_FIND_FRAMEWORK=last -DCMAKE_STRIP=/nix/store/a57856fs4m8ir6vlv14h3gq3sv9aq2lb-binutils-2.31.1/bin/strip -DCMAKE_RANLIB=/nix/store/a57856fs4m8ir6vlv14h3gq3sv9aq2lb-binutils-2.31.1/bin/ranlib -DCMAKE_AR=/nix/store/a57856fs4m8ir6vlv14h3gq3sv9aq2lb-binutils-2.31.1/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/xl963aynx8r48p0aqn85vmmxywyr4ckx-feedbackd-v0.0.0+git20200420
CMake Error: The source directory "/build/source" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
builder for '/nix/store/llm9jqrmrxjai94zmixjabkqp58fcpdx-feedbackd-v0.0.0+git20200420.drv' failed with exit code 1
```

Reply:
Are you sure `cmake` is needed? Meson supports it for finding dependencies but typically, `pkg-config` will be sufficient.

What happens if you remove `cmake`? Alternately, if it is really needed, you can use `dontUseCmakeConfigure = true;` to prevent it beating Meson.

Reply:
Removing 'cmake' in this package seems to work :) I remember trying this in the other ones and failing. I'll try again and report the results